config/locales/forms/edit_forms/en.yml

```diff
@@ -28,6 +34,14 @@ en:
               people: "Main people"
               company_no: "Companies House number"
               operator_address: "Address"
+            values:
+              business_type:
+                charity: "Charity or trust"
+                limitedCompany: "Limited company"
+                limitedLiabilityPartnership: "Limited liability partnership"
+                localAuthority: "Local authority or public body"
+                partnership: "Partnership"
+                soleTrader: "Individual or sole trader"
```

Comment:
We really have to find the time to deal with making `business_type` an `enum`! This list is everywhere across the apps!